README.md

```diff
@@ -30,64 +30,78 @@ We suggest a couple of alternative ways of installation.
 
```

Comment:
GIF on line 19 can be centred

Reply:
And information about port into IntelliJ plugin should be reworded to something like `A port to an IntelliJ plugin is available on [VirtusLab/git-machete-intellij-plugin](http://github.com/VirtusLab/git-machete-intellij-plugin/) repository and is constantly being developed`

Reply:
+ add a fragment (`#cośtam`) to redirect directly to plugin's readme

Reply:
All done